odml/terminology.py

```diff
@@ -19,7 +19,7 @@
 from .tools.xmlparser import XMLReader
 
 
-REPOSITORY_BASE = 'http://portal.g-node.org/odml/terminologies'
+REPOSITORY_BASE = 'https://terminologies.g-node.org'
 REPOSITORY = '/'.join([REPOSITORY_BASE, 'v1.1', 'terminologies.xml'])
```

Comment:
Should we change this to use `urljoin()` as well, like in the templates module?

Reply:
I'd rather overhaul the `terminologies` page when the common functions of `templates` and `terminologies` are generalized.